apps/03TicTacToe.js

```diff
@@ -24,32 +24,79 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  if ((board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn)
+    || (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn)
+    || (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn)) {
+    return true;
+  }
 }
 
 function verticalWin() {
-  // Your code here
+  if ((board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn)
+    || (board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn)
+    || (board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn)) {
+    return true;
+  }
 }
 
 function diagonalWin() {
-  // Your code here
+  if ((board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn)
+    || (board[0][2] === playerTurn && board[1][1] === playerTurn && board[2][0] === playerTurn)) {
+    return true;
+  }
 }
 
 function checkForWin() {
-  // Your code here
+  if (horizontalWin() || verticalWin() || diagonalWin()) {
+    console.log('Player ' + playerTurn + ' Won!');
+    return true;
+  }
+}
+
+function isSpaceAvailable(row, column) {
+  return (board[row][column] === ' ');
+}
```

Comment:
Nice and succinct. Would the three functions above this also work like this (as in changing "if" to "return" and removing the "return true" expression)?